src/core_read.cpp

```diff
@@ -44,10 +45,17 @@ opcodetype ParseOpCode(const std::string& s)
             }
         }
     }
+    opcodetype Parse(const std::string& s) const {
+        auto it = mapOpNames.find(s);
+        if (it == mapOpNames.end()) throw std::runtime_error("script parse error: unknown opcode");
+        return it->second;
```

Comment:
```suggestion
        return mapOpNames.at(s);
```

Reply:
NACK; this is a change to the error that gets returned.